.github/workflows/ci.yaml

```diff
@@ -19,4 +19,20 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - name: Run unit tests
-        run: npm run testing
+        run: npm test
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+      - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+      - name: Install dependencies
+        run: npm ci
+      - name: Check code style
+        run: npm run lint
```

Comment:
_This configures a second [job](https://docs.github.com/en/actions/getting-started-with-github-actions/core-concepts-for-github-actions#job) for the `ci` workflow (defined by this YAML file)._

_After checking out code, this reads the support Node.js version as defined by `.nvmrc` and invokes a [workflow command](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter) to setup Node.js._

Reply:
Noted